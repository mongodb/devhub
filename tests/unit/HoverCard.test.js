import React from 'react';
import { shallow } from 'enzyme';
import HoverCard from '../../src/components/dev-hub/hover-card';

const getHoverCard = to =>
    shallow(
        <HoverCard
            alt="Image with first cluster"
            image="/images/firstcluster.png"
            to={to}
        >
            <p>Here is some content</p>
        </HoverCard>
    );

it('hover cards without a link render correctly', () => {
    const tree = getHoverCard();
    expect(tree).toMatchSnapshot();
});

it('hover cards with a link render correctly', () => {
    const tree = getHoverCard('/');
    expect(tree).toMatchSnapshot();
});
