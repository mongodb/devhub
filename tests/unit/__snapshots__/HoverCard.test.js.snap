// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hover cards with a link render correctly 1`] = `
<ContentWrapper
  tabIndex="0"
  to="/"
>
  <Image
    alt="Image with first cluster"
    loading="lazy"
    src="/images/firstcluster.png"
  />
  <HoverContent>
    <p>
      Here is some content
    </p>
  </HoverContent>
</ContentWrapper>
`;

exports[`hover cards without a link render correctly 1`] = `
<RoundedContainer
  tabIndex="0"
>
  <Image
    alt="Image with first cluster"
    loading="lazy"
    src="/images/firstcluster.png"
  />
  <HoverContent>
    <p>
      Here is some content
    </p>
  </HoverContent>
</RoundedContainer>
`;
